### PR TITLE
Dev/issue 13530 b5 themes uppy data params title

### PR DESCRIPTION
**Pull request by sbreker**
This commit updates the Uppy data elements to 'kebab-case'. 

A bug was introduced when porting this script to B5 whereby image title
updates were not being saved because the assignment of this.$element
was changed. Revert saving $(element) to this.$element and change the 
data node to use a different class property.

### CODE REVIEW

File: plugins/arDominionB5Plugin/js/multiFileUpload.js [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
   function MultiFileUpload (element)
   {
-    this.$element = $(element).find('div.multiFileUpload');
+    this.$multiFileUpload = $(element).find('div.multiFileUpload');
+    this.$element = $(element);
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/js/multiFileUpload.js]: I don't understand why do you need two vars here ... based on line 386, you already have the correct element and line 8 should be enough to use `this.$element` for everything ... right?

**Review by jraddaoui — COMMENTED** [review]
Thanks @sbreker!

File: plugins/arDominionB5Plugin/js/multiFileUpload.js (hunk 1 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/js/multiFileUpload.js]: I have fixed this underlying issue allowing me to remove the second variable.

**Comment by sbreker** [comment]
Thanks for your review @jraddaoui!  I have pushed a new commit that better addresses the root issues affecting saving the titles correctly.

The selector in the document ready function was not selecting correctly due to kebab-case issues it seems. I have renamed this. Secondly, I have moved the data elements to a higher div on the page so that the full form is selected - the js needs access to buttons AND the data elements.

File: plugins/arDominionB5Plugin/js/multiFileUpload.js [hunk 2]
```diff
@@ -384,7 +383,8 @@
 
   $(function()
   {
-    var $node = $('.multiFileUpload');
+    var $node = $('.multifileupload-form');
+    //console.log($node.data('multifileupload-i18n-some-files-failed-error'));
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/js/multiFileUpload.js]: Ups ...

**Review by jraddaoui — APPROVED** [review]
Awesome, thanks @sbreker!

File: plugins/arDominionB5Plugin/js/multiFileUpload.js (hunk 2 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/js/multiFileUpload.js]: Removed...

**Pull request merged**